### PR TITLE
fix: unhandled typerror [no-ticket]

### PR DESCRIPTION
**Pull request by filfreire**

### CODE REVIEW

File: packages/insomnia/src/main/analytics.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export async function trackSegmentEvent(
   properties?: Record<string, any>,
 ) {
   const settings = await models.settings.get();
-  const { accountId } = await models.userSession.get();
+  const { accountId = '' } = await models.userSession.get();
```
Comment (gatzjames) [inline packages/insomnia/src/main/analytics.ts]:
```suggestion
  const userSession = await models.userSession.get();
```

File: packages/insomnia/src/main/analytics.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export async function trackSegmentEvent(
...
 
   const allowAnalytics = settings.enableAnalytics || accountId;
```
Comment (gatzjames) [inline packages/insomnia/src/main/analytics.ts]:
```suggestion
  const allowAnalytics = settings.enableAnalytics || userSession.accountId;
```

File: packages/insomnia/src/main/analytics.ts [hunk 3]
```diff
@@ -49,9 +49,9 @@ export async function trackSegmentEvent(
   properties?: Record<string, any>,
 ) {
   const settings = await models.settings.get();
-  const { accountId } = await models.userSession.get();
+  const userSession = await models.userSession.get();
 
-  const allowAnalytics = settings.enableAnalytics || accountId;
+  const allowAnalytics = settings.enableAnalytics || userSession.accountId;
```
Comment (gatzjames) [inline packages/insomnia/src/main/analytics.ts:54]:
```suggestion
  const allowAnalytics = settings.enableAnalytics || userSession?.accountId;
```

**Review by gatzjames — COMMENTED** [review]
TS skills

**Review by jackkav — APPROVED** [review]

**Pull request merged**